package.json

```diff
@@ -54,28 +54,29 @@
         "clipboard": "^2.0.11",
         "cookie-parser": "^1.4.6",
         "crypto-js": "^4.1.1",
-        "csv-parse": "^5.3.5",
+        "csv-parse": "^5.3.6",
+        "csv-stringify": "^6.2.4",
```

Comment:
👀 is this a new dep?

Reply:
Seems like an old dependency removed in #7284 . Should probably be removed here.
```suggestion
```

Reply:
Good catch, thanks, applied.